clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -471,9 +547,9 @@ synchronized void requestEpochBumpForPartition(TopicPartition tp) {
         this.partitionsToRewriteSequences.add(tp);
     }
 
-    private void bumpIdempotentProducerEpoch() {
+    private void bumpIdempotentProducerEpoch(@SuppressWarnings("SameParameterValue") CallingThread callingThread) {
```

Comment:
why do we need this suppression?

Reply:
Strictly speaking, it isn't needed, so I removed it.